File: 2917-count-pairs-whose-sum-is-less-than-target/count-pairs-whose-sum-is-less-than-target.cpp
class Solution {
public:
    int countPairs(vector<int>& nums, int target) {
        int n = nums.size(), pairs = 0, i, j;
        
        for (i = 0; i < n; i++)
            for (j = i + 1; j < n; j++)
                if (nums[i] + nums[j] < target) pairs++;

        return pairs;
    }
};

File: 2917-count-pairs-whose-sum-is-less-than-target/count-pairs-whose-sum-is-less-than-target.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        n = len(nums)
        pairs = 0

        for i in range(n):
            for j in range(i + 1, n):
                if nums[i] + nums[j] < target:
                    pairs += 1

        return pairs